crates/editor/src/element.rs

```diff
@@ -737,6 +744,7 @@ impl EditorElement {
         cx.with_z_index(1, |cx| {
             for (ix, fold_indicator) in layout.fold_indicators.drain(..).enumerate() {
                 if let Some(fold_indicator) = fold_indicator {
+                    assert!(gutter_settings.buttons);
```

Comment:
These should be `debug_assert!`s so that they don't crash in production.

Reply:
Ah, I forgot there were different asserts.